src/tests/ftest/nvme/enospace.py

```diff
@@ -409,9 +409,9 @@ def test_enospace_no_aggregation(self):
             time.sleep(60)
             print(pool_usage)
             #SCM pool size should be released (some still be used for system)
-            #Pool SCM free % should not be less than 50%
-            if pool_usage['scm'] > 55:
-                self.fail('SCM pool used percentage should be < 55, instead {}'.
+            #Pool SCM free % should not be less than 62%
+            if pool_usage['scm'] > 62:
+                self.fail('SCM pool used percentage should be < 62, instead {}'.
```

Comment:
Is this just "moving the target", or does it actually fix the issue? I.e. how do we determine 62%? And what happens when 62% no longer works?

Reply:
Thank you for the review and comment.
Per comment from JIRA ticket, we decided to increase the SCM pool usage percentage for the test.
I push this PR with 62% and see if test pass.

Reply:
How did you come up with 62? You might want to explain it in the comment and the commit message. Also, you might want to update the title because it says 56.17 instead of 62.

Reply:
I agree with @shimizukko - a comment would be useful.

Reply:
done